### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by sbrugman** [dylan-profiler/tangled-up-in-unicode#4]
```python
from tangled_up_in_unicode import script

character = "􀉆"
script(character)
```

Error:
```
  File "c:\anaconda3\lib\site-packages\tangled_up_in_unicode\tangled_up_in_unicode.py", line 413136, in script
    key_end = end_keys[insertion_point]
IndexError: list index out of range
```

_Originally posted by @vgroshev in https://github.com/pandas-profiling/pandas-profiling/issues/707_

**Comment by sbrugman** [comment]
Fixed in 0.0.7

**Comment by vgroshev** [comment]
Good afternoon! Is package update planning for pypi.org?
